nodejs/week1/MealWebsite/src/backend/app.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const app = express();
+const meals = require("./routes/meals.js");
+const cheapMeals = require("./routes/cheap-meals.js");
+const largeMeals = require("./routes/large-meals.js");
+const randomMeal = require("./routes/randommeal.js");
+const reservations = require("./routes/reservations.js");
+const randomReservation = require("./routes/reservation.js");
+
+app.get("/", (request, response) => {
+  response.send(`Page not found`);
+});
+
+app.get("/meals", (request, response) => {
+  response.send(meals);
+});
+
+app.get("/cheapMeals", (request, response) => {
```

Comment:
`"/cheapMeals"` ➡️  `"/cheap-meals"`

Avoid capital letters in urls, use dashes to split the words :)